src/main/java/org/jitsi/protocol/xmpp/util/SourceGroup.java

```diff
@@ -81,6 +82,19 @@ public SourceGroup(SourceGroupPacketExtension group)
     }
 
     /**
+     * Creates new instance of <tt>SourceGroup</tt>.
+     * @param semantics the group's semantics
+     * @param sources a {@link List} of the group's
+     *        {@link SourcePacketExtension}s
+     */
+    public SourceGroup(String semantics, List<SourcePacketExtension> sources)
```

Comment:
this seems to overlap a lot with the existing constructor (this takes in the fields of a `SourceGroupPacketExtension` whereas the other one just takes in a `SourceGroupPacketExtension` directly, is there a need for both?

Reply:
Yes, we need because it wraps the logic of creating new SourcePacketExtension. Which first needs to create new instance, set semantics and then set extensions. If you would need to use that in any other place this will be duplicating this logic.